bodhi/server/models.py

```diff
@@ -1000,9 +1000,12 @@ def get_pkg_committers_from_pagure(self):
         """
         pagure_url = config.get('pagure_url')
         # Pagure uses plural names for its namespaces such as "rpms" except for
-        # container. Flatpaks build directly from the 'modules' namespace
+        # container. Flatpaks were moved from 'modules' to 'flatpaks' - hence
+        # a config setting.
         if self.type.name == 'container':
             namespace = self.type.name
+        elif self.type.name == 'flatpak':
```

Comment:
Since this is an Enum, we can compare it directly:

```suggestion
        elif self.type == ContentType.flatpak:
```

Since there's another one above this, I'll just file a new PR to fix this and the other one together.